doc/fileexplorer.rst

```diff
@@ -1,28 +1,79 @@
-:orphan:
+#####
+Files 
+#####
 
-#############
-File Explorer
-#############
-
-The **File Explorer** pane is a built-in filesystem and directory browser allowing you to view and filter files and their properties, open them with the :doc:`editor` or an external tool, and perform common management tasks (varies by platform).
+The **Files** pane is a built-in filesystem and directory browser that allows you to view and filter files according to their type and extension.
+You can also open files with the :doc:`editor` or an external tool, and perform some operations.
 
 .. image:: images/file_explorer/file_explorer_standard.png
    :alt: Spyder File Explorer panel, showing a tree view of files and metadata
 
 |
 
 
-==========================
-Performing file operations
-==========================
 
-You can use context menus to access a number of functions on a per-file and per-directory basis, including running scripts, creating, renaming, moving, editing or deleting files, performing basic operations with ``git``, and (on Windows) opening an external file explorer.
+===============
+File Operations
+===============
+
+To browse between the files in your system, you can use the arrows in the top of the pane.
+You can expand/contract the folders displayed in the pane to deploy the files and subfolders hierarchically. 
+Double cliking a folder will open it showing the files on it and changing your working directory.
+
+[GIF showing browsing files and showing changing the working directory]
+
+To open a file in the editor from the file pane, double click over its name.
+If you right click over it, you will see a context menu that allows you to access a number of functions including running scripts, creating, renaming, moving, deleting and showing files in your computer's file manager.
+
+[Screenshot showing context menu]
+
+You can also copy and paste one or several files in the folder where you are located in the pane, or copy their absolute and relative's path in the clipboard to paste it as text.
+
+[GIF showing copy path and pasting it in console]
+
+Additionally, the Files pane allows you to perform basic operations with ``git`` like commiting the changes of a file and browsing the repository from which the file is part of.
+
+[GIF showing commiting change of file and then showing repo]
+
+
+
+============
+Options Menu
+============
+
+The options menu in the top right of the Find pane offers several ways to customize how your files are displayed.
+
+By default, the option Show all files is activated, which displays all your files in your working directory without filtering. 
+However, if you deactivate this option, you can filter the files that the pane displays according to their extensions by clicking the corresponding option.
```

Comment:
```suggestion
However, if you deactivate this, the pane will only show files matching the customizable patterns listed under :guilabel:`Edit filename filters...`.
```

Describe this much more clearly and explicitly; I was really confused by this sentence myself.

Reply:
Not sure if customizable patterns makes it clearer. Also this content is based on 4.x where the menu is different so this option you are describing doesn't exist.

Reply:
Thanks for the heads up; I didn't know there would be substantial UI changes and feature additions beyond just visual tweaks and solving bugs. I updated the above suggestion accordingly to reflect the change.

I see; I played around with it a bit but wasn't sure of a clearer and more understandable way to describe it that was accurate and not misleading. What might you suggest? The issue with the previous descriptor, "according to their extensions" is that they can actually match anywhere in the filename (in fact, several of the default patterns don't involve an extension at all), as well as duplicative of the guilabel text. @ccordoba12 your ideas here would be helpful as well.

As a sidenote, after testing it I have a few minor followup fixes I can help implement, for which I'll open an issue for and tag you two on.